students/mihwa-yang/users/views.py

```diff
@@ -1 +1,34 @@
-# Create your views here.
+import json
+
+from django.http  import JsonResponse
+from django.views import View
+from django.db    import IntegrityError
+
+from users.models import User
+
+class SignUpView(View): 
+    def post(self, request):
+        data  = json.loads(request.body)
+
+        if 'password' not in data.keys() or 'email' not in data.keys():
+            return JsonResponse({"MESSAGE": "KEY ERROR"}, status=400)
+
+        if data['email'].count('@') == 0 or data['email'].count('.') == 0:
+            return JsonResponse({"MESSAGE": "EMAIL FORM ERROR"}, status=400)
+        
+        password_length = len(data['password'])
+        if password_length < 8:
```

Comment:
```suggestion
        PASSWORD_LENGTH = 8
        if data['password'] < PASSWORD_LENGTH:
```
제가 의도한 상수화는 위와 같았습니다~

Reply:
아!!!!!!! 하면서도 긴가민가 했었는데..! 
경훈님 감사합니다 수정했습니다 👍